### PR TITLE
Update some favicons in 5.x 2022-07-24

### PR DESCRIPTION
**Pull request by dodmi**

### CODE REVIEW

File: data/favicon.json [hunk 1]
```diff
@@ -188,6 +188,8 @@
 	"instagram.com": "instagram.com.ico",
 	"instantsystems.se": "instantsystems.se.png",
 	"intesasanpaolo.com": "intesasanpaolo.com.ico",
+	"inxmail.com": "inxmail.com.ico",
+	"inxserver.com": "inxmail.com.ico",
```
Comment (lieser) [inline data/favicon.json:192]: Is this for mails what are coming from inxmail itself? Or for mails (e.g. newsletters) they send and sign on the behalf of overs?

At least for `inxserver.com` it is probably the latter.

And if it is the latter I do not want to add them to the favicon. I think signing newsletters just by the marketing company that is sending them is a confusing practice that should be discouraged from.

File: data/favicon.json [hunk 2]
```diff
@@ -324,6 +327,7 @@
 	"videolan.org": "videolan.org.ico",
 	"vistaprint.it": "vistaprint.it.png",
 	"voelkner.de": "voelkner.de.ico",
+	"watchlistmailings.de": "ing.de.png",
```
Comment (lieser) [inline data/favicon.json:330]: What is the relationship between `watchlistmailings.de` and `ing.de`?
Could not find any with a quick search, so I don't think maisl signed by `watchlistmailings.de` shoudl show the favicon of `ing.de`.

**Review by lieser — CHANGES_REQUESTED** [review]
Thanks for updating the rules, but I have a few questions about the favicons.

File: data/favicon.json (hunk 2 above)
Comment (dodmi) [reply data/favicon.json:330]: Ing is using watchlistmailings.de as mail domain for it's "Watchlist limit alerts":
https://www.ing.de/wertpapiere/watchlist-funktionen/

File: data/favicon.json (hunk 1 above)
Comment (dodmi) [reply data/favicon.json:192]: It's for newsletters, they send. I'll remove it then...

**Review by lieser — APPROVED** [review]

**Pull request merged**